### DISCUSSION TITLE
Is it possible immediately Sync a ArgoCD app the moment its updated via commit?

### DISCUSSION THREAD

**Discussion by iamraj007** [argoproj/argo-cd#18918] [Q&A] [answered]
So a new or existing Argo App should get updated immediately as soon as it have a new commit and as a exception not wait for 3 min (180 sec)

Is there any annotations,  syncPolicy or hook we can configure on Argo Application to make it work ?

**Comment by ChristianCiach** [comment, marked as answer]
Yes, this is what Webhooks are for: https://argo-cd.readthedocs.io/en/stable/operator-manual/webhook/

**Comment by vibhorsharma9999** [comment]
Or you can have a look at 
https://github.com/argoproj/argo-cd/blob/2d6ce088acd4fb29271ffb6f6023dbb27594d59b/docs/operator-manual/argocd-cm.yaml#L279-L282
